src/components/UI/Inputs/Select/index.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+  FormField,
+  FormFieldProps,
+  Select as Input,
+  ThemeContext,
+  ThemeType,
+} from 'grommet';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import { css } from 'styled-components';
+
+import SearchInput from './SearchInput';
+
+const customTheme: ThemeType = {
```

Comment:
Handy little trick to be able to customize nested components that can't be customized through props. These styles only apply to this component and its children.